department-of-veterans-affairs/abd-vro
# Update and rename vro-request.md to vro-request.yml

Trying to turn this into a form per GH instructions here: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

## What was the problem?
The .md request form template I created turned out kinda messy to use because it opens directly in edit mode, meaning instead of my nicely formatted markdown, users would see a wall of plain text with markdown syntax. I'm trying to use the yaml form option to make the content more readable.

Associated tickets or Slack threads:
- [slack thread](https://dsva.slack.com/archives/C04QLHM9LR0/p1700180106648889)

## How does this fix it?[^1]
I'm hoping this displays the content with embedded form fields, but I don't know how to preview it, so... 🤷‍♀️

## How to test this PR
- I don't know! Is there a way to get a display preview of a .yml file? 


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
